src/app/public/modules/summary-action-bar/summary-action-bar.component.ts

```diff
@@ -61,6 +61,9 @@ let nextId = 0;
   templateUrl: './summary-action-bar.component.html',
   styleUrls: ['./summary-action-bar.component.scss'],
   animations: [skyAnimationSlide],
+  providers: [
+    SkySummaryActionBarAdapterService
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```